build.gradle

```diff
@@ -1,11 +1,11 @@
 buildscript {
     repositories {
         jcenter()
-        google()
+        maven { url 'https://maven.google.com' }
```

Comment:
@samtstern Any reason we added the more verbose version of the repo instead of keeping `google()`?